packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -94,7 +94,7 @@ export const mapProjectToApplicationContextProject = (
 };
 
 type TApplicationContextEnvironment = ApplicationWindow['app'];
-type TApplicationContext<AdditionalEnvironmentProperties extends {}> = {
+export type TApplicationContext<AdditionalEnvironmentProperties extends {}> = {
```

Comment:
Exporting so that it can be used from other packages

Reply:
While reading: we actually do verify that when using withApplicationContext, that what is passed as a generic for the return type is actually something that is available? Just curious/a  thought.

Reply:
Hmm what do you mean? The `AdditionalEnvironmentProperties` part?

Reply:
Sort of. What I thought of isn’t possible though I think. I hoped that if we habe conext props and additional props. That whenever using some of them through a hook/hoc they are at least a subset of them.